tests/components/lcn/test_services.py

```diff
@@ -134,6 +135,21 @@ async def test_service_relays(
 
     control_relays.assert_awaited_with(relay_states)
 
+    # wrong states string
```

Comment:
We could have a separate parameterized `test_service_invalid_state_error` that would get the service and method to patch as a params, since this is repeated trough the various service tests. 
No need to do it in this PR. Its fine to leave it to a follow up.